src/components/organisms/AssetActions/Consume.tsx

```diff
@@ -118,7 +130,13 @@ export default function Consume({
           <File file={file} />
         </div>
         <div className={styles.pricewrapper}>
-          <Price ddo={ddo} conversion />
+          {isConsumable ? (
+            <Price ddo={ddo} conversion />
+          ) : (
+            <div className={styles.help}>
+              There is not enough liquidity in the pool to consume the data set.
```

Comment:
We don't use 'consume' in the UI so far, and if we use 'buy' instead it also makes it less confusing in the case of low liquidity/has datatokens

```suggestion
              There is not enough liquidity in the pool to buy this data set.
```